src/bufr_interface.f90

```diff
@@ -1,11 +1,14 @@
 !> @file
-!> @authors Ronald Mclaren
+!> @brief Enable a number of Fortran BUFRLIB functions to be called
+!>        from C and C++ application programs.
+!>
+!> @author Ronald Mclaren
 !> @date 2020-07-29
 !>
```

Comment:
What you need for modules, is two docblocks, one for the file, one for the module. Author tags on both. @brief is required for the file but not the module. Like this:

```
!> @file 
!! @brief A brief description of the file.
!! @author Freddy Fortran

!> This module contains some great stuff. ...
!> @author Freddy Fortran
module mod1
```

Reply:
OK, I can fix that as part of my next PR, unless you'd prefer that I go ahead and push up another commit on this one(?)

Reply:
I just went ahead and pushed up another commit to fix this.

Reply:
Fix as you see fit. We are figuring out some new doxygen stuff over on UFS_UTILS, and one is that programs and modules require separate doxygen docblocks for the file. So two docblocks in at the top of such files.

We are making a lot of progress filling in documentation for UFS_UTILS, which has (or had) a scary amount of undocumented code.